sdk/include/ndk/psfuncs.h

```diff
@@ -256,6 +256,20 @@ PsChargeProcessNonPagedPoolQuota(
     _In_ SIZE_T    Amount
 );
 
+NTKERNELAPI
+NTSTATUS
+NTAPI
+PsChargeCurrentProcessNonPagedPoolQuota(
```

Comment:
I doubt these functions exist (and therefore exported) on Windows (contrary to the PsChargeProcessNonPagedPoolQuota), so they should definitively **not be** added to the NDK.

Reply:
In hindsight, this was strange of me. Probably wasn't thinking straight.. Will fix!